presto-cli/src/main/java/io/prestosql/cli/CsvPrinter.java

```diff
@@ -31,13 +31,30 @@
 
     private boolean needHeader;
 
-    public CsvPrinter(List<String> fieldNames, Writer writer, boolean header)
+    public enum CsvOutputFormat
+    {
+        HEADER_WITH_QUOTES(true, true),
+        HEADER_WITHOUT_QUOTES(true, false),
+        NO_HEADER_WITH_QUOTES(false, true),
+        NO_HEADER_WITHOUT_QUOTES(false, false);
+
+        private boolean header;
+        private boolean quote;
+
+        CsvOutputFormat(boolean header, boolean quote)
+        {
+            this.header = header;
+            this.quote = quote;
+        }
+    }
+
+    public CsvPrinter(List<String> fieldNames, Writer writer, CsvOutputFormat csvOutputFormat)
     {
         requireNonNull(fieldNames, "fieldNames is null");
         requireNonNull(writer, "writer is null");
         this.fieldNames = ImmutableList.copyOf(fieldNames);
-        this.writer = new CSVWriter(writer);
-        this.needHeader = header;
+        this.writer = csvOutputFormat.quote ? new CSVWriter(writer) : new CSVWriter(writer, CSVWriter.DEFAULT_SEPARATOR, CSVWriter.NO_QUOTE_CHARACTER);
+        this.needHeader = csvOutputFormat.header;
```

Comment:
nit: you could provide two commits. First that introduces an enum (refactoring) and second that adds new options.